library/general/src/lib/ui/greasemonkey.rb

```diff
@@ -86,6 +86,12 @@ def VStackFrames(old)
     end
     module_function :VStackFrames
 
+    # @param old [Yast::Term]
+    # @return    [Yast::Term]
+    # @example
+    #   term(:FrameWithMarginBox, "Title", "arg1", "arg2")
+    #      ->
+    #   Frame("Title", MarginBox(1.45, 0.45, "arg1", "arg2"))
```

Comment:
if only use case is reusing code in yast2-partitioner, that why it is in yast2 where it should serve all modules? That is why I am more strict regarding API and other stuff, because one module scope it is ok, if there are some legacy stuff, but if we add it to yast2 itself, it means we have to be more careful in future when we want to improve it.